glean-core/src/lib.rs

```diff
@@ -992,6 +1012,12 @@ pub fn get_timestamp_ms() -> u64 {
     zeitstempel::now() / NANOS_PER_MILLI
 }
 
+/// Instructs the Metrics Ping Scheduler's thread to exit cleanly.
+/// If Glean was configured with `use_core_mps: false`, this has no effect.
+pub fn cancel_metrics_ping_scheduler() {
```

Comment:
Right now this is not called anywhere.
Should this be called in RLB's `shutdown` function?